akka-http/src/main/scala/akka/http/javadsl/server/directives/PathDirectives.scala

```diff
@@ -134,7 +134,7 @@ abstract class PathDirectives extends MiscDirectives {
    * }}}
    *
    * For further information, refer to:
-   * [[http://googlewebmastercentral.blogspot.de/2010/04/to-slash-or-not-to-slash.html]]
+   * @see "http://googlewebmastercentral.blogspot.de/2010/04/to-slash-or-not-to-slash.html"
```

Comment:
Are you sure this results in valid scaladoc and javadoc?
I'd think this breaks scaladoc